File: Project/Client/MyApplicationProject/MyApplication/src/main/java/com/example/myapplication/domain_objects/Rating.java
package com.example.myapplication.domain_objects;

/**
 * Created by Michal on 19/02/14.
 */
public class Rating {

    private int RatingId;
    private int UserId;
    private User TargetUser;
    private User FromUser;
    private String LeftOnDate;
    private int Score;
    private String Feedback;
    private int RatingContext;

    public int getRatingId() {
        return RatingId;
    }

    public int getUserId() {
        return UserId;
    }

    public User getTargetUser() {
        return TargetUser;
    }

    public User getFromUser() {
        return FromUser;
    }

    public int getScore() {
        return Score;
    }

    public String getLeftOnDate() {
        return LeftOnDate;
    }

    public String getFeedback() {
        return Feedback;
    }

    public int getRatingContext() {
        return RatingContext;
    }
}

File: Project/WebService/DomainObjects/Domains/Rating.cs
// --------------------------------------------------------------------------------------------------------------------
// <copyright file="Rating.cs" company="">
//   
// </copyright>
// <summary>
//   Defines the Rating type.
// </summary>
// --------------------------------------------------------------------------------------------------------------------

namespace DomainObjects.Domains
{
    using System;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Runtime.Serialization;

    using DomainObjects.Constants;

    /// <summary>
    /// The rating.
    /// </summary>
    [DataContract]
    public class Rating
    {
        /// <summary>
        /// Gets or sets the rating id.
        /// </summary>
        [DataMember]
        [ConcurrencyCheck]
        public int RatingId { get; set; }

        /// <summary>
        /// Gets or sets the user id.
        /// </summary>
        [DataMember]
        public int UserId { get; set; }

        /// <summary>
        /// Gets or sets the user.
        /// </summary>
        public User TargetUser { get; set; }

        /// <summary>
        /// Gets or sets the from user name.
        /// </summary>
        [DataMember]
        public User FromUser { get; set; }

        /// <summary>
        /// Gets or sets the left on date.
        /// </summary>
        [DataMember]
        [DataType(DataType.Date)]
        [Column(TypeName = "DateTime2")]
        public DateTime LeftOnDate { get; set; }

        /// <summary>
        /// Gets or sets the score.
        /// </summary>
        [DataMember]
        public int Score { get; set; }

        /// <summary>
        /// Gets or sets the feedback.
        /// </summary>
        [DataMember]
        public string Feedback { get; set; }
    }
}
